docs/decisions/0006-middleware-support-for-http-clients.rst

```diff
@@ -0,0 +1,40 @@
+middleware for HTTP client
+==========================================
+
+Status
+------
+
+
+
+Context
+-------
+
+We currently expose a HTTP client(axios instance) via ``getAuthenticatedHttpClient`` and ``getHttpClient`` used in
+our MFEs to make HTTP requests. Since edX uses Django for its backend services, the MFEs have to handle snake-cased <-> camelCase
+conversions manually. The 'axios-case-converter' package provides a middleware that would handle the conversion automatically. In the
```

Comment:
Is it worth calling out where/how the conversions take place (i.e., outgoing vs incoming data)?

Reply:
I was going to write a separate how-to doc for this.